tvbwidgets/ui/phase_plane_widget.py

```diff
@@ -645,6 +646,14 @@ def update_axis_sliders(self, _):
 
     def _add_model_selector(self):
         models = {model.__name__: model for model in models_module.ModelsEnum.get_base_model_subclasses()}
+
+        filtered_modules = {}
+        for name in models.keys():
```

Comment:
This method would be more efficient if you put this new `for` loop together with the one above at line 648.
```
models = {}
for model in models_module.ModelsEnum.get_base_model_subclasses():
      if model.__nvar >= 2:
             models[model.__name__] = model
```